client/js/lounge.js

```diff
@@ -448,6 +452,12 @@ $(function() {
 				.find(".unread-marker")
 				.appendTo(container);
 		}
+		if (data.msg.type === "channel_list") {
+			// Delay by 1 second so the browser has time to render all channels
+			$(target + " .chat").stop().delay(1000).animate({
+				scrollTop: 0
+			});
+		}
```

Comment:
I'm a bit concerned about this hardcoded 1 second. What does it accomplish?
I tried `/list` on Freenode just now (on v2.2.2, not your PR) and it never loads, tab actually hangs for a long time, and I'm running a fairly powerful laptop (though the server itself is on a RPi).

So yeah, not convinced about this, but maybe I'm missing something.